sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -58,34 +59,35 @@ export interface CertificateAttributes extends Attributes {
-    listDeletedCertificates(options?: KeyVaultClientGetDeletedCertificatesOptionalParams): PagedAsyncIterableIterator<DeletedCertificate, DeletedCertificate[]>;
-    mergeCertificate(certificateName: string, x509Certificates: Uint8Array[], options?: RequestOptionsBase): Promise<Certificate>;
-    purgeDeletedCertificate(certificateName: string, options?: RequestOptionsBase): Promise<null>;
-    recoverDeletedCertificate(certificateName: string, options?: RequestOptionsBase): Promise<Certificate>;
-    restoreCertificate(certificateBackup: Uint8Array, options?: RequestOptionsBase): Promise<Certificate>;
-    setCertificateContacts(contacts: Contact[], options?: RequestOptionsBase): Promise<Contacts>;
-    setCertificateIssuer(issuerName: string, provider: string, options?: KeyVaultClientSetCertificateIssuerOptionalParams): Promise<CertificateIssuer>;
-    updateCertificate(certificateName: string, version: string, options?: KeyVaultClientUpdateCertificateOptionalParams): Promise<Certificate>;
-    updateCertificateIssuer(issuerName: string, options?: KeyVaultClientUpdateCertificateIssuerOptionalParams): Promise<CertificateIssuer>;
-    updateCertificatePolicy(certificateName: string, policy: CertificatePolicy, options?: RequestOptionsBase): Promise<CertificatePolicy>;
+    backupCertificate(certificateName: string, options?: BackupCertificateOptions): Promise<BackupCertificateResult>;
+    cancelCertificateOperation(certificateName: string, options?: CancelCertificateOperationOptions): Promise<CertificateOperation>;
+    createCertificate(certificateName: string, certificatePolicy: CertificatePolicy, options?: CreateCertificateOptions): Promise<KeyVaultCertificate>;
+    deleteCertificate(certificateName: string, options?: DeleteCertificateOptions): Promise<DeletedCertificate>;
+    deleteCertificateContacts(options?: DeleteCertificateContactsOptions): Promise<Contacts>;
+    deleteCertificateIssuer(issuerName: string, options?: DeleteCertificateIssuerOptions): Promise<CertificateIssuer>;
+    deleteCertificateOperation(certificateName: string, options?: DeleteCertificateOperationOptions): Promise<CertificateOperation>;
+    getCertificate(certificateName: string, version: string, options?: GetCertificateOptions): Promise<KeyVaultCertificate>;
+    getCertificateContacts(options?: GetCertificateContactsOptions): Promise<Contacts>;
+    getCertificateIssuer(issuerName: string, options?: GetCertificateIssuerOptions): Promise<CertificateIssuer>;
+    getCertificateOperation(certificateName: string, options?: GetCertificateOperationOptions): Promise<CertificateOperation>;
+    getCertificatePolicy(certificateName: string, options?: GetCertificatePolicyOptions): Promise<CertificatePolicy>;
+    getCertificateWithPolicy(certificateName: string, options?: GetCertificateWithPolicyOptions): Promise<KeyVaultCertificate>;
```

Comment:
This should be renamed GetCertificate and GetCertificate should be renamed GetCertificateVersion.

Reply:
Couldn't do this today. Please check the updated description of this PR.

Reply:
This has been handled in @jonathandturner's commit https://github.com/Azure/azure-sdk-for-js/pull/5991/commits/1ef22d22baf363462e791bd9064546c1208a0bdf